src/components/ux-forms/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Manifest should be updated from boilerplate to reflect each data property used by the component